lti_consumer/migrations/0018_auto_20231225_0556.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.23 on 2023-12-25 10:56
```

Comment:
@shadinaif would you mind renaming on this migration file to something more readable similar to [other migrations](https://github.com/openedx/xblock-lti-consumer/tree/454346fb3ea0c3888b4dbb269594f2412d70a127/lti_consumer/migrations)?